### DISCUSSION TITLE
Emacs 28 with Native Compilation

### DISCUSSION THREAD

**Discussion by bobbyrullo** [Homebrew/discussions#3165] [Casks]
Hi there! Glad to see you guys have updated the Emacs cask to 28. However, one of the big features for that version is native compilation, and Brew's Cask was not compiled with that option it seems. 

I looked a little bit further and see what you guys are getting the dmg from https://emacsformacosx.com/, so maybe it has to be fixed with them, but I figured I'd start here to see if you were aware and had a relationship with those folks.

Thanks so much,

Bobby

**Comment by SMillerDev** [comment]
Nope, casks are all just running the default upstream installer. If there's anything that needs special compilation you'll need to contact upstream.

**Comment by bobbyrullo** [comment]
Gotcha Sean. I'll see what I can do with those guys.

Thanks again,

Bobby

**Comment by carlocab** [comment]
You might also want to consider using one of these external taps to install `emacs`, assuming the `emacs` formula doesn't suit your needs:

https://github.com/railwaycat/homebrew-emacsmacport
https://github.com/d12frosted/homebrew-emacs-plus

Note that these taps are not affiliated with Homebrew (in the same way that the maintainer of the source of the Emacs cask is not).

**Comment by z-aki** [comment]
https://github.com/z-aki/homebrew-tap

this only differs from brew upstream emacs in the native-compilation flag. Everything else is the same. macOS-15 and 26 are supported for now.

https://github.com/Homebrew/homebrew-core/pull/98548#issuecomment-1091025084
